common/src/main/java/de/florianmichael/rclasses/common/memory/Memory.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * This file is part of RClasses - https://github.com/FlorianMichael/RClasses
+ * Copyright (C) 2023 FlorianMichael/EnZaXD and contributors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package de.florianmichael.rclasses.common.memory;
+
+import java.lang.reflect.Field;
+
+/**
+ * This class represents functions to work with native memory
+ */
+public class Memory {
+    /**
+     * The unsafe instance.
+     */
+    public final static sun.misc.Unsafe UNSAFE = getUnsafe();
+
+    /**
+     * @return the unsafe instance
+     */
+    public static sun.misc.Unsafe getUnsafe() {
+        try {
+            for (Field field : sun.misc.Unsafe.class.getDeclaredFields()) {
+                if (field.getType().equals(sun.misc.Unsafe.class)) {
+                    field.setAccessible(true);
+                    return (sun.misc.Unsafe) field.get(null);
+                }
+            }
+        } catch (Throwable ignored) {
+        }
+        throw new IllegalStateException("Unable to get Unsafe instance");
+    }
+
+    /**
+     * Allocates memory
+     * @param bytes The number of bytes
+     * @return the pointer address
+     */
+    public static long allocate(long bytes) {
```

Comment:
What's the point of this method? Just make the unsafe field accessible for users so they can do whatever they want with it.

Reply:
I added this because in newer java version, the `sun.misc.Unsafe` will get removed, so I added this to not introduce breaking changes

Reply:
Is the class really for removal? I don't think they will actually remove that as it would break a lot of Software depending on it, however, as soon as they remove this class we would have to find a replacement for those methods which doesn't seem like a good approach to me.